enterprise/server/remote_execution/vfs/vfs_unix.go

```diff
@@ -112,11 +114,22 @@ func (r *latencyRecorder) Add(name string, dt time.Duration) {
 	stats.timeSpent += dt
 }
 
+type vfsTaskLogger struct {
+	vfs *VFS
+}
+
+func (v *vfsTaskLogger) Write(p []byte) (n int, err error) {
+	log.CtxDebugf(v.vfs.getRPCContext(), "[go-fuse] "+strings.TrimRight(string(p), "\n"))
+	return len(p), nil
+}
+
 func (vfs *VFS) Mount() error {
 	if vfs.server != nil {
 		return status.FailedPreconditionError("VFS already mounted")
 	}
 
+	flog := golog.New(&vfsTaskLogger{vfs}, "", 0)
```

Comment:
why do we want to use the builtin logger instead of our own here?

(Comment could help the reader here)